nautobot/core/management/commands/generate_test_data.py

```diff
@@ -292,6 +293,7 @@ def _create_batch(some_factory, count, description="", **kwargs):
         )
         _create_batch(MetadataChoiceFactory, 20)
         _create_batch(ObjectChangeFactory, 100)
+        _create_batch(ObjectMetadataFactory, 20)
```

Comment:
It'd probably be more "realistic" to create a larger number of these, maybe 100 for now?